src/hooks/useGetModules.js

```diff
@@ -9,14 +9,69 @@ const useGetModules = (key) => {
   const objectPromises = { public1: getPublic1, public2: getPublic2, writes: getAllWriters };
```

Comment:
Apparently, there's an error in deploy is related to 'getAllWriters'
![image](https://user-images.githubusercontent.com/73669701/176067982-97625172-bad6-41a3-b4f3-e4c9e7798146.png)